diff --git a/node_modules/sigma/sigma.d.ts b/node_modules/sigma/sigma.d.ts
index 7ca945c..8374b26 100644
--- a/node_modules/sigma/sigma.d.ts
+++ b/node_modules/sigma/sigma.d.ts
@@ -217,6 +217,12 @@ export default class Sigma extends TypedEventEmitter<SigmaEvents> {
      * @return {Camera}
      */
     getCamera(): Camera;
+    /**
+     * Method returning the DOM container element.
+     *
+     * @return {HTMLElement}
+     */
+    getContainer(): HTMLElement;
     /**
      * Method returning the renderer's graph.
      *
diff --git a/node_modules/sigma/sigma.js b/node_modules/sigma/sigma.js
index af60ee4..e1ac8c7 100644
--- a/node_modules/sigma/sigma.js
+++ b/node_modules/sigma/sigma.js
@@ -983,6 +983,14 @@ var Sigma = /** @class */ (function (_super) {
     Sigma.prototype.getCamera = function () {
         return this.camera;
     };
+    /**
+     * Method returning the DOM container element.
+     *
+     * @return {HTMLElement}
+     */
+    Sigma.prototype.getContainer = function () {
+        return this.container;
+    };
     /**
      * Method returning the renderer's graph.
      *
